### ISSUE TITLE
Enforce all expects to have an actual case to test

### ISSUE THREAD

**Issue by diego-G** [lo1tuma/eslint-plugin-mocha#183]
As highlighted in the issue https://github.com/LiskHQ/lisk/issues/2480, there is no rule to enforce all the expects to have an actual case to test. This can lead to false positives due to mistakes.

Below the _wrong_ case that should raise a warning:
```node
describe('calculateFee', () => {
	it('should return the constant fee', () => {
		return expect(block.calculateFee(blockData).isEqualTo(FEES.SEND));
	});
});
```
The _corrected_ solution:
```node
describe('calculateFee', () => {
	it('should return the constant fee', () => {
		return expect(block.calculateFee(blockData).isEqualTo(FEES.SEND)).to.be.true;
	});
});
```

**Comment by lo1tuma** [comment]
Mocha doesn’t come with any assertion library. The examples you provided look like `expect` from [`chai`](http://www.chaijs.com/).
So maybe you want to have a look at chai specific eslint plugins like [`eslint-plugin-chai-expect`](https://www.npmjs.com/package/eslint-plugin-chai-expect).
